src/localization/buttonStrings.json

```diff
@@ -77,7 +78,8 @@
     "customer_data": "Données Clients",
     "factory_reset": "Réinitialiser",
     "check_connection": "Vérifiez la connexion",
-    "connection_status": "Connexion réussie"
+    "connection_status": "Connexion réussie",
```

Comment:
@arjunSussol if the keys does not exist then translation would fallback to default language keys. Which means we do not need to add keys (english keys) in french or other language... it can be translated it time but for now please remove them.

Reply:
Done!